[2025-10-06T22:56:55.266+0000] {processor.py:153} INFO - Started process (PID=38) to work on /opt/airflow/dags/airflow.py
[2025-10-06T22:56:55.269+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T22:56:55.271+0000] {logging_mixin.py:137} INFO - [2025-10-06T22:56:55.271+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T22:56:55.287+0000] {logging_mixin.py:137} INFO - [2025-10-06T22:56:55.283+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 4, in <module>
    from airflow.providers.standard.operators.python import PythonOperator
ModuleNotFoundError: No module named 'airflow.providers.standard'
[2025-10-06T22:56:55.288+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T22:56:55.307+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.045 seconds
[2025-10-06T22:57:25.633+0000] {processor.py:153} INFO - Started process (PID=86) to work on /opt/airflow/dags/airflow.py
[2025-10-06T22:57:25.634+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T22:57:25.634+0000] {logging_mixin.py:137} INFO - [2025-10-06T22:57:25.634+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T22:57:25.639+0000] {logging_mixin.py:137} INFO - [2025-10-06T22:57:25.638+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 4, in <module>
    from airflow.providers.standard.operators.python import PythonOperator
ModuleNotFoundError: No module named 'airflow.providers.standard'
[2025-10-06T22:57:25.640+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T22:57:25.654+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.023 seconds
[2025-10-06T22:57:56.040+0000] {processor.py:153} INFO - Started process (PID=119) to work on /opt/airflow/dags/airflow.py
[2025-10-06T22:57:56.041+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T22:57:56.042+0000] {logging_mixin.py:137} INFO - [2025-10-06T22:57:56.042+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T22:57:56.053+0000] {logging_mixin.py:137} INFO - [2025-10-06T22:57:56.048+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 4, in <module>
    from airflow.providers.standard.operators.python import PythonOperator
ModuleNotFoundError: No module named 'airflow.providers.standard'
[2025-10-06T22:57:56.054+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T22:57:56.092+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.063 seconds
[2025-10-06T22:58:26.333+0000] {processor.py:153} INFO - Started process (PID=168) to work on /opt/airflow/dags/airflow.py
[2025-10-06T22:58:26.334+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T22:58:26.334+0000] {logging_mixin.py:137} INFO - [2025-10-06T22:58:26.334+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T22:58:26.341+0000] {logging_mixin.py:137} INFO - [2025-10-06T22:58:26.340+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 4, in <module>
    from airflow.providers.standard.operators.python import PythonOperator
ModuleNotFoundError: No module named 'airflow.providers.standard'
[2025-10-06T22:58:26.342+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T22:58:26.364+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.034 seconds
[2025-10-06T22:58:56.645+0000] {processor.py:153} INFO - Started process (PID=207) to work on /opt/airflow/dags/airflow.py
[2025-10-06T22:58:56.647+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T22:58:56.650+0000] {logging_mixin.py:137} INFO - [2025-10-06T22:58:56.650+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T22:58:56.659+0000] {logging_mixin.py:137} INFO - [2025-10-06T22:58:56.657+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 4, in <module>
    from airflow.providers.standard.operators.python import PythonOperator
ModuleNotFoundError: No module named 'airflow.providers.standard'
[2025-10-06T22:58:56.660+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T22:58:56.686+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.046 seconds
[2025-10-06T22:59:27.032+0000] {processor.py:153} INFO - Started process (PID=249) to work on /opt/airflow/dags/airflow.py
[2025-10-06T22:59:27.034+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T22:59:27.035+0000] {logging_mixin.py:137} INFO - [2025-10-06T22:59:27.035+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T22:59:27.041+0000] {logging_mixin.py:137} INFO - [2025-10-06T22:59:27.040+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 4, in <module>
    from airflow.providers.standard.operators.python import PythonOperator
ModuleNotFoundError: No module named 'airflow.providers.standard'
[2025-10-06T22:59:27.042+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T22:59:27.064+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.044 seconds
[2025-10-06T22:59:38.190+0000] {processor.py:153} INFO - Started process (PID=266) to work on /opt/airflow/dags/airflow.py
[2025-10-06T22:59:38.192+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T22:59:38.193+0000] {logging_mixin.py:137} INFO - [2025-10-06T22:59:38.193+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T22:59:39.058+0000] {logging_mixin.py:137} INFO - [2025-10-06T22:59:39.057+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 7, in <module>
    from src.lab import load_data, data_preprocessing, build_save_model, load_model_elbow
  File "/opt/airflow/dags/src/lab.py", line 2, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T22:59:39.058+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T22:59:39.067+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.883 seconds
[2025-10-06T22:59:43.140+0000] {processor.py:153} INFO - Started process (PID=268) to work on /opt/airflow/dags/airflow.py
[2025-10-06T22:59:43.142+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T22:59:43.143+0000] {logging_mixin.py:137} INFO - [2025-10-06T22:59:43.143+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T22:59:43.443+0000] {logging_mixin.py:137} INFO - [2025-10-06T22:59:43.441+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 7, in <module>
    from src.lab import load_data, data_preprocessing, build_save_model, load_model_elbow
  File "/opt/airflow/dags/src/lab.py", line 2, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T22:59:43.443+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T22:59:43.455+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.320 seconds
[2025-10-06T23:00:13.578+0000] {processor.py:153} INFO - Started process (PID=318) to work on /opt/airflow/dags/airflow.py
[2025-10-06T23:00:13.581+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T23:00:13.582+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:00:13.582+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T23:00:13.962+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:00:13.961+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 7, in <module>
    from src.lab import load_data, data_preprocessing, build_save_model, load_model_elbow
  File "/opt/airflow/dags/src/lab.py", line 2, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T23:00:13.962+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T23:00:13.972+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.401 seconds
[2025-10-06T23:00:41.855+0000] {processor.py:153} INFO - Started process (PID=37) to work on /opt/airflow/dags/airflow.py
[2025-10-06T23:00:41.858+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T23:00:41.862+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:00:41.862+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T23:00:42.085+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:00:42.083+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 7, in <module>
    from src.lab import load_data, data_preprocessing, build_save_model, load_model_elbow
  File "/opt/airflow/dags/src/lab.py", line 2, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T23:00:42.085+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T23:00:42.101+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.248 seconds
[2025-10-06T23:01:12.234+0000] {processor.py:153} INFO - Started process (PID=86) to work on /opt/airflow/dags/airflow.py
[2025-10-06T23:01:12.235+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T23:01:12.235+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:01:12.235+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T23:01:12.500+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:01:12.496+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 7, in <module>
    from src.lab import load_data, data_preprocessing, build_save_model, load_model_elbow
  File "/opt/airflow/dags/src/lab.py", line 2, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T23:01:12.500+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T23:01:12.518+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.289 seconds
[2025-10-06T23:01:42.759+0000] {processor.py:153} INFO - Started process (PID=121) to work on /opt/airflow/dags/airflow.py
[2025-10-06T23:01:42.760+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T23:01:42.761+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:01:42.760+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T23:01:42.960+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:01:42.955+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 7, in <module>
    from src.lab import load_data, data_preprocessing, build_save_model, load_model_elbow
  File "/opt/airflow/dags/src/lab.py", line 2, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T23:01:42.961+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T23:01:43.052+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.304 seconds
[2025-10-06T23:02:21.017+0000] {processor.py:153} INFO - Started process (PID=36) to work on /opt/airflow/dags/airflow.py
[2025-10-06T23:02:21.018+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T23:02:21.020+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:02:21.020+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T23:02:21.416+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:02:21.412+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 7, in <module>
    from src.lab import load_data, data_preprocessing, build_save_model, load_model_elbow
  File "/opt/airflow/dags/src/lab.py", line 2, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T23:02:21.417+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T23:02:21.475+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.463 seconds
[2025-10-06T23:02:51.793+0000] {processor.py:153} INFO - Started process (PID=70) to work on /opt/airflow/dags/airflow.py
[2025-10-06T23:02:51.794+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T23:02:51.795+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:02:51.795+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T23:02:52.106+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:02:52.099+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 7, in <module>
    from src.lab import load_data, data_preprocessing, build_save_model, load_model_elbow
  File "/opt/airflow/dags/src/lab.py", line 2, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T23:02:52.107+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T23:02:52.136+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.349 seconds
[2025-10-06T23:03:10.330+0000] {processor.py:153} INFO - Started process (PID=37) to work on /opt/airflow/dags/airflow.py
[2025-10-06T23:03:10.334+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T23:03:10.335+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:03:10.335+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T23:03:10.608+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:03:10.600+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 7, in <module>
    from src.lab import load_data, data_preprocessing, build_save_model, load_model_elbow
  File "/opt/airflow/dags/src/lab.py", line 2, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T23:03:10.609+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T23:03:10.636+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.309 seconds
[2025-10-06T23:03:41.157+0000] {processor.py:153} INFO - Started process (PID=71) to work on /opt/airflow/dags/airflow.py
[2025-10-06T23:03:41.158+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T23:03:41.159+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:03:41.159+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T23:03:41.440+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:03:41.435+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 7, in <module>
    from src.lab import load_data, data_preprocessing, build_save_model, load_model_elbow
  File "/opt/airflow/dags/src/lab.py", line 2, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T23:03:41.441+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T23:03:41.462+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.312 seconds
[2025-10-06T23:04:07.849+0000] {processor.py:153} INFO - Started process (PID=37) to work on /opt/airflow/dags/airflow.py
[2025-10-06T23:04:07.853+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T23:04:07.854+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:04:07.853+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T23:04:08.106+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:04:08.104+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 7, in <module>
    from src.lab import load_data, data_preprocessing, build_save_model, load_model_elbow
  File "/opt/airflow/dags/src/lab.py", line 2, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T23:04:08.106+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T23:04:08.128+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.282 seconds
[2025-10-06T23:04:38.165+0000] {processor.py:153} INFO - Started process (PID=80) to work on /opt/airflow/dags/airflow.py
[2025-10-06T23:04:38.166+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T23:04:38.166+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:04:38.166+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T23:04:38.742+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:04:38.732+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 7, in <module>
    from src.lab import load_data, data_preprocessing, build_save_model, load_model_elbow
  File "/opt/airflow/dags/src/lab.py", line 2, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T23:04:38.742+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T23:04:38.776+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.614 seconds
[2025-10-06T23:05:09.169+0000] {processor.py:153} INFO - Started process (PID=120) to work on /opt/airflow/dags/airflow.py
[2025-10-06T23:05:09.171+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T23:05:09.172+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:05:09.171+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T23:05:09.355+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:05:09.350+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 7, in <module>
    from src.lab import load_data, data_preprocessing, build_save_model, load_model_elbow
  File "/opt/airflow/dags/src/lab.py", line 2, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T23:05:09.355+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T23:05:09.369+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.204 seconds
[2025-10-06T23:05:39.505+0000] {processor.py:153} INFO - Started process (PID=170) to work on /opt/airflow/dags/airflow.py
[2025-10-06T23:05:39.506+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T23:05:39.507+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:05:39.506+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T23:05:39.664+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:05:39.661+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 7, in <module>
    from src.lab import load_data, data_preprocessing, build_save_model, load_model_elbow
  File "/opt/airflow/dags/src/lab.py", line 2, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T23:05:39.664+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T23:05:39.677+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.178 seconds
[2025-10-06T23:06:10.055+0000] {processor.py:153} INFO - Started process (PID=213) to work on /opt/airflow/dags/airflow.py
[2025-10-06T23:06:10.056+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T23:06:10.057+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:06:10.057+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T23:06:10.251+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:06:10.247+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 7, in <module>
    from src.lab import load_data, data_preprocessing, build_save_model, load_model_elbow
  File "/opt/airflow/dags/src/lab.py", line 2, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T23:06:10.251+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T23:06:10.270+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.216 seconds
[2025-10-06T23:06:40.333+0000] {processor.py:153} INFO - Started process (PID=254) to work on /opt/airflow/dags/airflow.py
[2025-10-06T23:06:40.335+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T23:06:40.336+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:06:40.336+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T23:06:40.529+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:06:40.525+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 7, in <module>
    from src.lab import load_data, data_preprocessing, build_save_model, load_model_elbow
  File "/opt/airflow/dags/src/lab.py", line 2, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T23:06:40.529+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T23:06:40.544+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.218 seconds
[2025-10-06T23:07:10.602+0000] {processor.py:153} INFO - Started process (PID=304) to work on /opt/airflow/dags/airflow.py
[2025-10-06T23:07:10.603+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T23:07:10.604+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:07:10.604+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T23:07:10.808+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:07:10.805+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 7, in <module>
    from src.lab import load_data, data_preprocessing, build_save_model, load_model_elbow
  File "/opt/airflow/dags/src/lab.py", line 2, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T23:07:10.809+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T23:07:10.824+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.227 seconds
[2025-10-06T23:07:41.196+0000] {processor.py:153} INFO - Started process (PID=339) to work on /opt/airflow/dags/airflow.py
[2025-10-06T23:07:41.198+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T23:07:41.199+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:07:41.199+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T23:07:41.361+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:07:41.357+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 7, in <module>
    from src.lab import load_data, data_preprocessing, build_save_model, load_model_elbow
  File "/opt/airflow/dags/src/lab.py", line 2, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T23:07:41.362+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T23:07:41.377+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.184 seconds
[2025-10-06T23:08:33.372+0000] {processor.py:153} INFO - Started process (PID=37) to work on /opt/airflow/dags/airflow.py
[2025-10-06T23:08:33.382+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T23:08:33.382+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:08:33.382+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T23:08:33.604+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:08:33.597+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 7, in <module>
    from src.lab import load_data, data_preprocessing, build_save_model, load_model_elbow
  File "/opt/airflow/dags/src/lab.py", line 2, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T23:08:33.605+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T23:08:33.624+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.273 seconds
[2025-10-06T23:09:03.806+0000] {processor.py:153} INFO - Started process (PID=80) to work on /opt/airflow/dags/airflow.py
[2025-10-06T23:09:03.807+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T23:09:03.809+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:09:03.809+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T23:09:04.369+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:09:04.363+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 7, in <module>
    from src.lab import load_data, data_preprocessing, build_save_model, load_model_elbow
  File "/opt/airflow/dags/src/lab.py", line 2, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T23:09:04.370+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T23:09:04.416+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.617 seconds
[2025-10-06T23:09:34.791+0000] {processor.py:153} INFO - Started process (PID=121) to work on /opt/airflow/dags/airflow.py
[2025-10-06T23:09:34.794+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T23:09:34.794+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:09:34.794+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T23:09:35.066+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:09:35.061+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 7, in <module>
    from src.lab import load_data, data_preprocessing, build_save_model, load_model_elbow
  File "/opt/airflow/dags/src/lab.py", line 2, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T23:09:35.067+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T23:09:35.085+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.298 seconds
[2025-10-06T23:10:05.391+0000] {processor.py:153} INFO - Started process (PID=171) to work on /opt/airflow/dags/airflow.py
[2025-10-06T23:10:05.394+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T23:10:05.397+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:10:05.397+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T23:10:05.642+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:10:05.637+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 7, in <module>
    from src.lab import load_data, data_preprocessing, build_save_model, load_model_elbow
  File "/opt/airflow/dags/src/lab.py", line 2, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T23:10:05.642+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T23:10:05.680+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.296 seconds
[2025-10-06T23:10:36.111+0000] {processor.py:153} INFO - Started process (PID=214) to work on /opt/airflow/dags/airflow.py
[2025-10-06T23:10:36.113+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T23:10:36.114+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:10:36.114+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T23:10:36.310+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:10:36.303+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 7, in <module>
    from src.lab import load_data, data_preprocessing, build_save_model, load_model_elbow
  File "/opt/airflow/dags/src/lab.py", line 2, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T23:10:36.310+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T23:10:36.334+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.228 seconds
[2025-10-06T23:11:06.477+0000] {processor.py:153} INFO - Started process (PID=255) to work on /opt/airflow/dags/airflow.py
[2025-10-06T23:11:06.479+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T23:11:06.479+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:11:06.479+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T23:11:06.701+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:11:06.696+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 7, in <module>
    from src.lab import load_data, data_preprocessing, build_save_model, load_model_elbow
  File "/opt/airflow/dags/src/lab.py", line 2, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T23:11:06.701+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T23:11:06.727+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.255 seconds
[2025-10-06T23:11:36.975+0000] {processor.py:153} INFO - Started process (PID=305) to work on /opt/airflow/dags/airflow.py
[2025-10-06T23:11:36.978+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T23:11:36.978+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:11:36.978+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T23:11:37.169+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:11:37.164+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 7, in <module>
    from src.lab import load_data, data_preprocessing, build_save_model, load_model_elbow
  File "/opt/airflow/dags/src/lab.py", line 2, in <module>
    from sklearn.preprocessing import MinMaxScaler
ModuleNotFoundError: No module named 'sklearn'
[2025-10-06T23:11:37.170+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T23:11:37.184+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.214 seconds
[2025-10-06T23:14:54.215+0000] {processor.py:153} INFO - Started process (PID=38) to work on /opt/airflow/dags/airflow.py
[2025-10-06T23:14:54.215+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T23:14:54.217+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:14:54.217+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T23:14:54.787+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:14:54.785+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 7, in <module>
    from src.lab import load_data, data_preprocessing, build_save_model, load_model_elbow
  File "/opt/airflow/dags/src/lab.py", line 4, in <module>
    from kneed import KneeLocator
ModuleNotFoundError: No module named 'kneed'
[2025-10-06T23:14:54.787+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T23:14:54.805+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.594 seconds
[2025-10-06T23:15:25.198+0000] {processor.py:153} INFO - Started process (PID=87) to work on /opt/airflow/dags/airflow.py
[2025-10-06T23:15:25.200+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T23:15:25.201+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:15:25.200+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T23:15:25.905+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:15:25.904+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 7, in <module>
    from src.lab import load_data, data_preprocessing, build_save_model, load_model_elbow
  File "/opt/airflow/dags/src/lab.py", line 4, in <module>
    from kneed import KneeLocator
ModuleNotFoundError: No module named 'kneed'
[2025-10-06T23:15:25.906+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T23:15:25.913+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.718 seconds
[2025-10-06T23:15:56.184+0000] {processor.py:153} INFO - Started process (PID=137) to work on /opt/airflow/dags/airflow.py
[2025-10-06T23:15:56.185+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T23:15:56.187+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:15:56.187+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T23:15:56.928+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:15:56.926+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 7, in <module>
    from src.lab import load_data, data_preprocessing, build_save_model, load_model_elbow
  File "/opt/airflow/dags/src/lab.py", line 4, in <module>
    from kneed import KneeLocator
ModuleNotFoundError: No module named 'kneed'
[2025-10-06T23:15:56.928+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T23:15:56.938+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.757 seconds
[2025-10-06T23:16:27.024+0000] {processor.py:153} INFO - Started process (PID=171) to work on /opt/airflow/dags/airflow.py
[2025-10-06T23:16:27.025+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T23:16:27.026+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:16:27.026+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T23:16:27.534+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:16:27.533+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 7, in <module>
    from src.lab import load_data, data_preprocessing, build_save_model, load_model_elbow
  File "/opt/airflow/dags/src/lab.py", line 4, in <module>
    from kneed import KneeLocator
ModuleNotFoundError: No module named 'kneed'
[2025-10-06T23:16:27.534+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T23:16:27.545+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 0.526 seconds
[2025-10-06T23:16:57.844+0000] {processor.py:153} INFO - Started process (PID=221) to work on /opt/airflow/dags/airflow.py
[2025-10-06T23:16:57.846+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T23:16:57.847+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:16:57.847+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T23:16:59.471+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:16:59.467+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/airflow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/airflow.py", line 7, in <module>
    from src.lab import load_data, data_preprocessing, build_save_model, load_model_elbow
  File "/opt/airflow/dags/src/lab.py", line 4, in <module>
    from kneed import KneeLocator
ModuleNotFoundError: No module named 'kneed'
[2025-10-06T23:16:59.471+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T23:16:59.494+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 1.659 seconds
[2025-10-06T23:17:41.441+0000] {processor.py:153} INFO - Started process (PID=37) to work on /opt/airflow/dags/airflow.py
[2025-10-06T23:17:41.444+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/airflow.py for tasks to queue
[2025-10-06T23:17:41.444+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:17:41.444+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/airflow.py
[2025-10-06T23:17:42.427+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:17:42.427+0000] {taskmixin.py:205} WARNING - Dependency <Task(PythonOperator): load_data_task>, data_preprocessing_task already registered for DAG: Airflow_Lab1
[2025-10-06T23:17:42.428+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:17:42.428+0000] {taskmixin.py:205} WARNING - Dependency <Task(PythonOperator): data_preprocessing_task>, load_data_task already registered for DAG: Airflow_Lab1
[2025-10-06T23:17:42.429+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:17:42.429+0000] {taskmixin.py:205} WARNING - Dependency <Task(PythonOperator): data_preprocessing_task>, build_save_model_task already registered for DAG: Airflow_Lab1
[2025-10-06T23:17:42.429+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:17:42.429+0000] {taskmixin.py:205} WARNING - Dependency <Task(PythonOperator): build_save_model_task>, data_preprocessing_task already registered for DAG: Airflow_Lab1
[2025-10-06T23:17:42.429+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:17:42.429+0000] {taskmixin.py:205} WARNING - Dependency <Task(PythonOperator): build_save_model_task>, load_model_task already registered for DAG: Airflow_Lab1
[2025-10-06T23:17:42.430+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:17:42.430+0000] {taskmixin.py:205} WARNING - Dependency <Task(PythonOperator): load_model_task>, build_save_model_task already registered for DAG: Airflow_Lab1
[2025-10-06T23:17:42.469+0000] {processor.py:753} INFO - DAG(s) dict_keys(['Airflow_Lab1']) retrieved from /opt/airflow/dags/airflow.py
[2025-10-06T23:17:42.734+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:17:42.734+0000] {manager.py:504} INFO - Created Permission View: can edit on DAG:Airflow_Lab1
[2025-10-06T23:17:42.739+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:17:42.739+0000] {manager.py:504} INFO - Created Permission View: can read on DAG:Airflow_Lab1
[2025-10-06T23:17:42.743+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:17:42.743+0000] {manager.py:504} INFO - Created Permission View: can delete on DAG:Airflow_Lab1
[2025-10-06T23:17:42.744+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:17:42.743+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2025-10-06T23:17:42.756+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:17:42.756+0000] {dag.py:2711} INFO - Creating ORM DAG for Airflow_Lab1
[2025-10-06T23:17:42.766+0000] {logging_mixin.py:137} INFO - [2025-10-06T23:17:42.766+0000] {dag.py:3441} INFO - Setting next_dagrun for Airflow_Lab1 to 2025-10-05T23:17:42.765979+00:00, run_after=2025-10-06T23:17:42.765979+00:00
[2025-10-06T23:17:42.783+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/airflow.py took 1.344 seconds
